### PR TITLE
Add AMOLED dark theme

### LINKED ISSUE #29: [FTR] AMOLED
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
An AMOLED theme, which many people have requested. Would prefer it to be a switch on the current night option (so that battery rules are still followed).

**Describe alternatives you've considered**
Currently, the default Android DayNight is used, but it's not AMOLED...

**Additional context**
(ufg)

### PR DESCRIPTION
**Pull request by ammargitham**
This will add a AMOLED option in the settings to be used as the dark theme instead of the default dark theme. Basically sets the background color as black in most places. Only dialogs, bottomsheets are kept as the default dark color for contrast.

Also, translations are required for the new setting added.

Fixes #29

### CODE REVIEW

File: app/src/main/java/awais/instagrabber/MainHelper.java [hunk 1]
```diff
@@ -400,23 +394,23 @@ public void onDrawerClosed(@NonNull final View drawerView, @MouseDrawer.EdgeGrav
...
+        //     final Bitmap bitmap = Bitmap.createBitmap(9, 9, Bitmap.Config.ARGB_8888);
+        //     final Canvas canvas = new Canvas();
+        //     canvas.setBitmap(bitmap);
+        //     background.draw(canvas);
+        //     color = bitmap.getPixel(4, 4);
+        //     if (!bitmap.isRecycled()) bitmap.recycle();
+        // }
+        // if (color == -1 || color == 0) color = resources.getBoolean(R.bool.isNight) ? 0xff212121 : 0xfff5f5f5;
+        // main.mainBinding.profileView.profileInfo.setBackgroundColor(color);
+        // if (!isBottomToolbar) main.mainBinding.toolbar.toolbar.setBackgroundColor(color);
```
Comment (ammargitham) [inline app/src/main/java/awais/instagrabber/MainHelper.java:413]: Not sure why this was being done here, copying the color from the appbar and applying it to the toolbar. as they can be set in xml/styles. I removed it and fixed the weird collapsing of the profile view toolbar which resulted after removing this code. Result is same as it was before change.

File: app/src/main/java/awais/instagrabber/activities/Main.java [hunk 2]
```diff
@@ -108,11 +107,6 @@ public void onClick(final View v) {
     private Stack<String> queriesStack;
     private DataBox.CookieModel cookieModel;
 
-    public Main() {
-        super();
-        Utils.changeTheme();
-    }
-
```
Comment (ammargitham) [inline app/src/main/java/awais/instagrabber/activities/Main.java:115]: Moved this to the parent activity.

File: app/src/main/java/awais/instagrabber/MainHelper.java (hunk 1 above)
Comment (austinhuang0131) [reply app/src/main/java/awais/instagrabber/MainHelper.java:413]: not really, I think this codes prevents this from happening

Screen Shot 2020-08-10 at 14 10 45

Comment (ammargitham) [reply app/src/main/java/awais/instagrabber/MainHelper.java:413]: Yeah. That's why added noscroll to the collapsing toolbar. As other that preventing the scroll nothing else was being achieved.

Comment (ammargitham) [reply app/src/main/java/awais/instagrabber/MainHelper.java:413]: Okay, I got the problem. Let me make some changes. I'll add it to this pull request.

**Comment by ammargitham** [comment]
Okay, so the commit https://github.com/austinhuang0131/instagrabber/pull/45/commits/31d02b3e1217179956d0304964d92f7da4d902a1 makes the entire Appbar scrollable. Is this behavior fine?

**Pull request merged**